server/src/main/java/io/crate/execution/engine/aggregation/impl/StandardDeviationAggregation.java

```diff
@@ -62,7 +62,7 @@
 
 public class StandardDeviationAggregation extends AggregationFunction<StandardDeviation, Double> {
 
-    public static final String NAME = "stddev";
+    public static final String[] NAMES = { "stddev", "stddev_pop" };
```

Comment:
```suggestion
    public static final List<String> NAMES = List.of("stddev", "stddev_pop");
```

Arrays are mutable - that should be avoided for constants

Reply:
thx, addressed.